ground/src/test/java/com/google/android/ground/ui/home/HomeScreenFragmentTest.kt

```diff
@@ -162,7 +162,6 @@ class HomeScreenFragmentTest : AbstractHomeScreenFragmentTest() {
     swipeUpDrawer()
 
     onView(withId(R.id.terms_of_service)).check(matches(isEnabled()))
-    onView(withId(R.id.version_text)).check(matches(isDisplayed()))
```

Comment:
Can we assert the build text in some other way?

Reply:
done